cluster-http/src/test/scala/akka/cluster/http/ClusterHttpApiSpec.scala

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.cluster.http
+
+import language.postfixOps
+import org.scalatest.WordSpecLike
+import org.scalatest.Matchers
+import akka.actor.Address
+import akka.cluster.ClusterEvent.CurrentClusterState
+import akka.cluster._
+import akka.http.scaladsl.model.{ FormData, StatusCodes }
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import org.mockito.ArgumentMatchers._
+import org.mockito.Mockito._
+
+import scala.collection.immutable._
+
+class ClusterHttpApiSpec extends WordSpecLike with Matchers with ScalatestRouteTest with ClusterHttpApiJsonProtocol {
+
+  "Cluster Http API" should {
+    "return list of members" when {
+      "calling GET /members" in {
+        val address1 = Address("akka", "Main", "hostname.com", 3311)
+        val address2 = Address("akka", "Main", "hostname2.com", 3311)
+        val address3 = Address("akka", "Main", "hostname3.com", 3311)
+
+        val uniqueAddress1 = UniqueAddress(address1, 1)
+        val uniqueAddress2 = UniqueAddress(address2, 2)
+
+        val clusterMember1 = Member(uniqueAddress1, Set())
+        val clusterMember2 = Member(uniqueAddress2, Set())
+        val currentClusterState = CurrentClusterState(SortedSet(clusterMember1, clusterMember2))
+
+        val unreachable = Map(
+          UniqueAddress(address3, 2) -> Set(uniqueAddress1, uniqueAddress2)
+        )
+
+        val mockedCluster = mock(classOf[Cluster])
```

Comment:
not big fan of mocking, but this might actually simplify the test and library here (compared to a full-blown multi-jvm test).